documentation/assemblies/cruise-control/assembly-cruise-control-concepts.adoc

```diff
@@ -3,11 +3,16 @@
 // master.adoc
 
 [id='cruise-control-concepts-{context}']
-= Cruise Control
+= Cruise Control for cluster rebalancing
 
-You can deploy {CruiseControlProject} and access a subset of its features through {ProductName} custom resources.
+You can deploy {CruiseControlProject} to your {ProductName} cluster and access its _cluster rebalancing_ feature through custom resources.
 
-Example YAML files for deploying Cruise Control and setting optimization goals are provided in `examples/cruise-control/`.
+Cruise Control is an open source system for automating Kafka operations, including monitoring cluster workload, rebalancing a cluster based on predefined constraints, and detecting and fixing anomalies, among others. 
```

Comment:
Do we want to list things we don't support here like anomaly detection? Maybe we could  be clear in this intro what is supported. _Strimzi currently supports the <> features of Cruise Control_

Reply:
Right. It's not that helpful to start with an explanation what we don't support.

I'll add a list of supported features as you suggest, and relegate the not supported features to a note.